spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/EnableAutoConfigurationImportSelectorTests.java

```diff
@@ -166,6 +167,13 @@ public void propertyOverrideSetToFalse() throws Exception {
 		assertThat(imports).isEmpty();
 	}
 
+	@Test(expected = IllegalStateException.class)
```

Comment:
We usually use the `ExpectedException` `@Rule` for this to make sure the exception is thrown exactly where it is supposed to.